include/experimental/debug.h

```diff
@@ -3,4 +3,39 @@
 
 #define BREAKPOINT __asm jr $ __endasm;
 
-#endif
\ No newline at end of file
+#include <string.h>
+#include <knightos/display.h>
+
+#define __DEBUG_TIMEOUT 100
+
+#define debug_hinted(/* SCREEN* */ screen, msg, /* char* */ hint, /* void(SCREEN*, unsigned char, unsigned char, msg) */ draw_func) \
```

Comment:
This really shouldn't be a macro. Macros should only be used if there's a real pressing reason, which there is not here.

Given the inefficient codegen, it's probably cheaper to generate a full function for it than to inline it everywhere - but i'd rather let the compiler's inliner decide.

Reply:
The reason why it is a macro is to have the line of code included in the debug message, and to allow custom draw functions if needed.

Reply:
Ahh, I see. I overlooked that the first time.